ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/management-security.j2

```diff
@@ -94,6 +94,42 @@
 {%             endif %}
 {%         endfor %}
 {%     endif %}
+{%     if management_security.password.policies is arista.avd.defined %}
+
+### Password Policies
+{%         for policy in management_security.password.policies %}
+#### Policy {{ policy.name }}
+
```

Comment:
Always place the blank line after the `if` and avoid blanks at the end of a section. This will ensure consistent spacing.